### DISCUSSION TITLE
Label in `captionLayout="dropdown"`

### DISCUSSION THREAD

**Discussion by Sparticuz** [gpbl/react-day-picker#2411] [General]
Should these lines exist if the `captionLayout` is set to "dropdown"? Or am I missing something?
![image](https://github.com/user-attachments/assets/0143b33c-18fc-472e-aa46-2ed9c1a0a27d)

https://github.com/gpbl/react-day-picker/blob/bb8eb927b066f5ad6c51b343b700a767de82e1b2/src/components/Dropdown.tsx#L54-L61

**Comment by gpbl** [comment]
> Should these lines exist if the `captionLayout` is set to "dropdown"

@Sparticuz yes this is the visual label and the chevron that are displayed to style the select element (which has an opacity set to 0). It's a way to style the select element, whose style is difficult to change.

Are you running into some issues with them?
